### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by brunotl**

### CODE REVIEW

File: Common/Source/Dialogs/dlgWayQuick.cpp [hunk 1]
```diff
@@ -132,7 +133,7 @@ short dlgWayQuickShowModal(void){
   if (WPLSEL.Format == LKW_CUP) {
         TCHAR ttmp[50];
         // and it is landable
-        if ((WPLSEL.Style>1) && (WPLSEL.Style<6) ) {
+        if ((WPLSEL.Style >= STYLE_AIRFIELDGRASS) && (WPLSEL.Style <= STYLE_AIRFIELDSOLID) ) {
```
Comment (pventafridda) [inline Common/Source/Dialogs/dlgWayQuick.cpp]: Why not the same >1 ? Style 1 is not landable. We should respect <= etc. , no?

**Comment by pventafridda** [comment]
Originally we were searching among the farvisibles, now we are searching going through the entire waypoint list. But we are no more inside draw thread, so probably it is not a problem, we can wait (less than before in any case, right?)

**Comment by brunotl** [comment]
right in V5 wait time are 2sec + search time
now it's only search time

File: Common/Source/Dialogs/dlgWayQuick.cpp (hunk 1 above)
Comment (brunotl) [inline Common/Source/Dialogs/dlgWayQuick.cpp]: no problem, it's more clear like that STYLE_AIRFIELDGRASS are first landable and STYLE_AIRFIELDSOLID are last landable.

**Pull request merged**